app/main/forms.py

```diff
@@ -2350,7 +2350,7 @@ class BrandingRequestForm(StripWhitespaceForm):
 class GovernmentIdentityLogoForm(StripWhitespaceForm):
     logo_text = GovukTextInputField(
         "Enter the text that will appear in your logo",
-        validators=[DataRequired("Cannot be empty")],
+        validators=[DataRequired("Enter the text that will appear in your logo")],
```

Comment:
Think the pattern now is to use the `NotifyDataRequired` class instead:
```suggestion
        validators=[NotifyDataRequired(thing="the text that will appear in your logo")],
```